libs/langchain-google-common/src/connection.ts

```diff
@@ -245,6 +288,19 @@ export abstract class GoogleAIConnection<
     }
   }
 
+  get computedLocation(): string {
+    switch (this.apiName) {
+      case "google":
+        return super.computedLocation;
```

Comment:
Why isn't this just `super.location`?

Reply:
`computedLocation` is typically only called from `location`. Keeping it calling the same method on the superclass keeps it clean and avoids additional logic processing.